src/mam/mod_mam_rdbms_arch.erl

```diff
@@ -172,46 +187,40 @@ retract_message(Host, #{archive_id := UserID,
                         direction := Dir,
                         packet := Packet}) ->
     case mod_mam_utils:get_retract_id(mod_mam, Host, Packet) of
-        none -> ok;
-        OriginIDToRetract -> retract_message(Host, UserID, LocJID, RemJID, OriginIDToRetract, Dir)
+        none ->
+            ok;
+        OriginIDToRetract ->
+            retract_message(Host, UserID, LocJID, RemJID, OriginIDToRetract, Dir)
```

Comment:
I don't see any improvement here. Why change it? I think it's OK to have one clause per line in simple cases. Easier to grep.